index.html

```diff
@@ -4,36 +4,44 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <script
-      src="https://kit.fontawesome.com/afb53e8513.js"
-      crossorigin="anonymous"
-    ></script>
+  
     <link rel="stylesheet" href="style.css" />
     <link rel="stylesheet" href="responsive.css" />
+    <!-- font awesome -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
     <title>brian nyachae's portfolio</title>
   </head>
   <body>
     <!--main section wrapper starts here-->
-    <div class="wrapper">
-      <header>
-        <nav>
-          <div class="content heading">
-            <div class="logo-container">
-              <div class="logo-item">
-                <a class="logo" href="index.html">My Logo</a>
-              </div>
-            </div>
 
-            <div class="menu-top">
-              <ul class="menu-item">
-                <li class="menu-links">Portfolio</li>
-                <li class="menu-links">About</li>
-                <li class="menu-links">Contact</li>
-              </ul>
+    <div class="wrapper">
+      <!-- mobile navigation section -->
+      <div class="mobile-nav">
```

Comment:
- [ ] The hamburger button and menu should follow the design from Figma, but I noticed that on mobile screens, the logo gets next to hamburger button, please make sure that logo and hamburger button are alight on the left and right side as shown in Figma:

Your design  | Figma design
------------- | -------------
<img width="292" alt="Screenshot 2022-04-06 at 13 51 12" src="https://user-images.githubusercontent.com/77038610/161948520-769d5c47-e41a-4829-acf2-7c64728d41d3.png"> | <img width="402" alt="Screenshot 2022-04-06 at 13 51 47" src="https://user-images.githubusercontent.com/77038610/161948703-d62d7dc2-f106-4c4b-95d9-accedf8e2f32.png">